portfolio/src/main/java/com/google/sps/servlets/DataServletCQC.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+
+@WebServlet("/my-CQCs")
+public class DataServletCQC extends HttpServlet { //Data Servlet for Comment Question Concern form in portfolio
```

Comment:
This comment would probably work better in the javadoc format described here:
https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line

So for example, this would be 

```
/* Data Servlet for Comment Question Concern form in portfolio. */
@WebServlet("/my-CQCs")
public class DataServletCQC extends HttpServlet {
```